cloud/deploys/staging_azure/setup

```diff
@@ -53,4 +53,4 @@ terraform \
 terraform \
   -chdir="${TEMPLATE_DIR}" \
   apply \
-  -var-file="${TF_VAR_FILE}"
+  -var-file="${TF_VAR_FILENAME}"
```

Comment:
i think this is wrong and also like we probably don't need this file. since the chdir changes to the directory the file has to be colocated

Reply:
we shouldn't pass `-var-file=`?

Reply:
no I think we don't need this setup bash file, since all the setup is being done with the python file. Also TF_VAR_FILENAME should just be a filename without the directory path before it, since chdir changes into the directory where the file is located

Reply:
removed